hrms/payroll/doctype/retention_bonus/retention_bonus.py

```diff
@@ -43,7 +43,7 @@ def on_submit(self):
 	def on_cancel(self):
 
 		additional_salary = self.get_additional_salary()
-		if self.additional_salary:
+		if additional_salary:
```

Comment:
`self.additional_salary` -> `additional_salary` will have to be updated in all the remaining places too in cancel function

Reply:
all 'self.additional_salary' convert to 'additional_salary' in cancel function